DelvUI/Helpers/Vector2Helper.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Numerics;
+using System.Text;
+using System.Threading.Tasks;
+using DelvUI.Interface;
+
+namespace DelvUI.Helpers
+{
+    class Vector2Helper
+    {
+        public static Vector2 ChangeX(Vector2 v, float x)
+        {
+            return new Vector2(x, v.Y);
+        }
+
+        public static Vector2 ChangeY(Vector2 v, float y)
+        {
+            return new Vector2(v.X, y);
+        }
+    }
+}
```

Comment:
I don't see this being used.
If so please remove.

Reply:
Unrelated but just in case it ever helps... You can also do vec2 + 1 and end up with the result of new Vector2(vec2.x + 1, vec2.y + 1)

Vector2.One and Vector2.Zero exist as do Vector2.UnitX and Vector2.UnitY

Vector2.UnitX * scale  would result in new Vector2(scale, 0)

Reply:
I removed it since it doesn't serve its purpose with the new config system in place.